xml/containers-support.xml

```diff
@@ -12,47 +12,63 @@
   <title>Compatibility and support plans</title>
  </info>
  <sect1 xml:id="sec-sle-container-support-matrix">
-  <title>&sle; container support matrix</title>
+  <title>Support information</title>
 
-  <para>
-   Consult the following support and compatibility matrix to make sure that the
-   desired host system and container combination is compatible and supported.
-  </para>
-
-  <informalfigure>
-   <mediaobject>
-    <imageobject role="fo">
-     <imagedata fileref="container_support_matrix.png" width="100%" format="PNG"/>
-    </imageobject>
-    <imageobject role="html">
-     <imagedata fileref="container_support_matrix.png" width="100%" format="PNG"/>
-    </imageobject>
-   </mediaobject>
-  </informalfigure>
+  <sect2 xml:id="sec-sle-container-non-sle-host-support">
+   <title>Support for &slsa; hosts</title>
+   <para>
+    Consult the following support and compatibility matrix to make sure that
+    the desired host system and container combination is compatible and
+    supported.
+   </para>
+   <informalfigure>
+    <mediaobject>
+     <imageobject role="fo">
+      <imagedata fileref="container_support_matrix.png" width="100%" format="PNG"/>
+     </imageobject>
+     <imageobject role="html">
+      <imagedata fileref="container_support_matrix.png" width="100%" format="PNG"/>
+     </imageobject>
+    </mediaobject>
```

Comment:
This figure lacks a description for visually impaired users, see https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-figure. As Jana pointed out in the last meeting, this lowers the SEO ranking.

Also, using a table instead of an image of a table is more accessible for visually impaired users. On the other hand, our stylesheet's support for tables is limited (no alignment or custom background colors). So we probably have to stick to tables (for now).